src/javaharness/WORKSPACE

```diff
@@ -46,6 +46,13 @@ maven_jar(
     artifact = "javax.inject:javax.inject:jar:sources:1"
 )
 
+maven_jar(
+    name = "com_google_guava_guava",
+    artifact = "com.google.guava:guava:18.0",
```

Comment:
guava is up to version 28.

Also there's an android version.

https://github.com/google/guava has the details.  I have not used the gradle version; so can't say how well it works.

Reply:
sorry, i should have added a description to the PR ..

i'll bump up the version, but otherwise i'm getting a 
```/java/arcs/api/CollectionProxy.java:3: error: package com.google.common.util.concurrent does not exist
import com.google.common.util.concurrent.ListenableFuture;
                                        ^
```
and i don't know what is missing.

Reply:
I found this might be useful: https://github.com/bazelbuild/rules_jvm_external